celeri.py

```diff
@@ -2249,7 +2249,7 @@ def test_okada_equals_cutde():
                 y_obs[i],
                 z_obs[i],
             ],  # (meters) observation point
-            -segment_locking_depth,  # (meters) depth of the fault origin
+            segment_locking_depth,  # (meters) depth of the fault origin
```

Comment:
Depth is measured in positive downwards for Okada.

Reply:
It looks like I also have change this in get_okada_displacements (near line 1040).  Thanks!